src/main/kotlin/apply/application/EvaluationTargetCsvService.kt

```diff
@@ -0,0 +1,88 @@
+import apply.utils.CsvGenerator
+import apply.utils.CsvRow
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVParser
+import org.apache.commons.csv.CSVRecord
+import org.springframework.stereotype.Service
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import javax.transaction.Transactional
+
+@Transactional
+@Service
+class EvaluationTargetCsvService(
+    private val evaluationTargetService: EvaluationTargetService,
+    private val evaluationItemRepository: EvaluationItemRepository,
+    private val csvGenerator: CsvGenerator
+) {
+    fun createTargetCsv(evaluationId: Long): ByteArrayInputStream {
+        val targets = evaluationTargetService.findAllByEvaluationIdAndKeyword(evaluationId)
+        val evaluationItems = evaluationItemRepository.findByEvaluationIdOrderByPosition(evaluationId)
+        val evaluationItemHeaders = evaluationItems.map { it.toHeader() }.toTypedArray()
+
+        val headerTitles = arrayOf(ID, NAME, EMAIL, STATUS, *evaluationItemHeaders)
+        val csvRows = targets.map { target ->
+            CsvRow(
+                target.id.toString(),
+                target.name,
+                target.email,
+                target.evaluationStatus.name,
+                *scores(target.answers, evaluationItems).toTypedArray()
+            )
+        }
+        return csvGenerator.generateBy(headerTitles, csvRows)
+    }
+
+    private fun EvaluationItem.toHeader(): String = "${this.title}(${this.maximumScore})"
+
+    private fun scores(
+        answers: List<EvaluationAnswerResponse>,
+        evaluationItems: List<EvaluationItem>
+    ): List<String> {
+        val answerScores = answers.associate { it.evaluationItemId to it.score.toString() }
+        return evaluationItems.map { answerScores[it.id] ?: "0" }
+    }
+
+    fun updateTarget(inputStream: InputStream, evaluationId: Long) {
+        val evaluationItems = evaluationItemRepository.findByEvaluationIdOrderByPosition(evaluationId)
+        inputStream.bufferedReader().use { reader ->
+            val csvParser = CSVParser(
+                reader,
+                CSVFormat.DEFAULT
+                    .withFirstRecordAsHeader()
+                    .withTrim()
+            )
+            for (csvRecord in csvParser) {
+                var targetId = csvRecord.get(ID)
+                var name = csvRecord.get(NAME)
+                var email = csvRecord.get(EMAIL)
+                var evaluationStatus = EvaluationStatus.valueOf(csvRecord.get(STATUS))
+                val evaluationAnswers = readEvaluationAnswers(evaluationItems, csvRecord)
+                // TODO: 평가 대상자 상태 업데이트 기능 구현
+            }
+        }
+    }
+
+    private fun readEvaluationAnswers(
+        evaluationItems: List<EvaluationItem>,
+        csvRecord: CSVRecord
+    ): List<EvaluationAnswer> = evaluationItems.map { evaluationItem ->
+        val score = csvRecord.get(evaluationItem.toHeader())
```

Comment:
람다가 중첩되지 않으니 `evaluationItem` 대신 `it`을 사용할 수 있습니다.

```suggestion
        val score = csvRecord.get(it.toHeader())
```